src/sinks/gcp/stackdriver/metrics/mod.rs

```diff
@@ -0,0 +1,20 @@
+// TODO: In order to correctly assert component specification compliance, we would have to do some more advanced mocking
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏We should have a doc comment here to describe in brief the sink.